### PR TITLE
Added sfx to pillars and grid

### PR DESCRIPTION
**Pull request by Omar-Shueb**

### CODE REVIEW

File: vr-wordle/src/Components/Pillars.js [hunk 1]
```diff
@@ -13,7 +13,11 @@ export default function Pillars(props) {
         guessIndex={3}
         setGuess={props.setGuess}
       />
-      <Column position={[0, 0, 0.7]} guessIndex={2} setGuess={props.setGuess} />
+      <Column
+        position={[0, -0.1, 1]}
```
Comment (Omar-Shueb) [inline vr-wordle/src/Components/Pillars.js:17]:
```suggestion
        position={[0, -0.1, 0.7]}
```

File: vr-wordle/src/App.js [hunk 2]
```diff
@@ -176,7 +170,7 @@ export default function App() {
         fade
       />
       <fog attach="fog" args={["#421700", 0, 100]} />
-      <PositionalAudio url="/wind.mp3" distance={1} autoplay loop />
+      <Wind />
     </VRCanvas>
```
Comment (tomw13) [inline vr-wordle/src/App.js:174]: What does this Wind component do?

File: vr-wordle/src/Components/Grid.js [hunk 3]
```diff
@@ -23,6 +24,19 @@ function StaticLetter(props) {
...
+  const buffer = useLoader(THREE.AudioLoader, "/blocks.mp3");
+
+  useEffect(() => {
+    sound.current.setBuffer(buffer);
+    sound.current.setRefDistance(1);
+    sound.current.setVolume(0.9);
+    camera.add(listener);
+    return () => camera.remove(listener);
+  });
+
```
Comment (tomw13) [inline vr-wordle/src/Components/Grid.js:39]: Really great work here

File: vr-wordle/src/Components/Grid.js [hunk 4]
```diff
@@ -33,7 +47,8 @@ function StaticLetter(props) {
 
   useFrame(() => {
     if (props.submitted && box.current.rotation.x < Math.PI / 2) {
-      box.current.rotation.x += 0.03 - props.index * 0.005;
```
Comment (tomw13) [inline vr-wordle/src/Components/Grid.js:36]: What is the box here? is this to mimic the effect of positional audio?

File: vr-wordle/src/App.js (hunk 2 above)
Comment (Omar-Shueb) [reply vr-wordle/src/App.js:174]: Adds wind sound effects to the scene, I wanted to componentize as much as possible

**Review by tomw13 — APPROVED** [review]

**Pull request merged**